packages/peerplays/lib/peerplays.js

```diff
@@ -107,6 +133,30 @@ export default class PPY extends Plugin {
     return ChainValidation.is_account_name(name);
   }
 
+  getImportableAccounts(keypair, network){
+		return new Promise((resolve, reject) => {
+      if(!keypair.username) {
+        console.log('no username');
```

Comment:
this is a bit confusing. It's missing data so we just spit a message into a console and `resolve(...)` instead of `reject(...)` or `throw new Error('...')`. Is there a reason for the reverse result of what actually occurred?